### ISSUE TITLE
Failure to return files without output_file_prefix

### ISSUE THREAD

**Issue by Medoalmasry** [replicate/cog#1265]
I don't have "A base URL to upload output files to. [src](https://github.com/replicate/cog/blob/main/docs/http.md)"
How can I possibly retrieve the file as a response?

I used the example [here](https://github.com/replicate/cog/blob/main/docs/python.md#file) 
```
from cog import BasePredictor, File, Input, Path
from PIL import Image

class Predictor(BasePredictor):
    def predict(self, source_image: File = Input(description="Image to enlarge")) -> File:
        pillow_img = Image.open(source_image)
        upscaled_image = do_some_processing(pillow_img)
        return File(upscaled_image)
```

Building is successful but the predict function produces the following error:-
`ⅹ /predictions call returned status 500`

**Comment by satani99** [comment]
Hi, @Medoalmasry I'm also getting the same error. Have you found any solution? Thanks

**Comment by manu-sapiens** [comment]
Indeed, File() does not seem to accept parameters. My approach is to return a Path instead of a File, but I can't say I'm having great success either. It would be nice to update the documentation for File with an example that works.

**Comment by markphelps** [comment]
This issue has been open for over two years with no recent activity or clear reproduction steps, so I'm closing it as stale. If this is still a problem in the latest version of Cog, please open a new issue with a minimal reproduction, your cog.yaml, and the full stack trace.
